bib/internal/setup/setup.go

```diff
@@ -53,6 +53,14 @@ func EnsureEnvironment() error {
 	if err := util.RunCmdSync("mount", "--bind", destPath, osbuildPath); err != nil {
 		return err
 	}
+
+	// And we also forcibly label the store to ensure we're not grabbing container labels
+	rootType := "system_u:object_r:root_t:s0"
```

Comment:
One thing I'd like to test as a followup; now that we have a policy in bib since https://github.com/osbuild/bootc-image-builder/pull/176 we can probably just do `chcon --reference /` here instead of hardcoding a context.  And similarly `chcon --reference=/usr/bin/rpm` instead of hardcoding install_t.